src/pages/updates/v10-migration/design.mdx

```diff
@@ -1,145 +1,147 @@
+
+</PageDescription>
+
 <AnchorLinks>
 
 <AnchorLink>IBM Design Language</AnchorLink>
 <AnchorLink>Design kit</AnchorLink>
 <AnchorLink>Elements</AnchorLink>
 <AnchorLink>Design migration strategy</AnchorLink>
-<AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
 ## IBM Design Language
 
-Carbon v10 delivers the new <a href="https://www.ibm.com/design/language/" target="_blank">IBM Design Language</a> to product teams. It represents a complete visual refresh of the system, delivering the ethos of IBM Design. With tooling and guidance on color, layout, typography, iconography, motion, patterns, and content, Carbon v10 is a modern, open-source framework for building digital products and experiences.
+Carbon v10 delivers the [IBM Design Language](https://www.ibm.com/design/language/) to product teams. It represents a complete visual refresh of the system, delivering the ethos of IBM Design. With tooling and guidance on color, layout, typography, iconography, motion, patterns, and content, Carbon v10 is a modern, open-source framework for building digital products and experiences.
 
-## Carbon v10 kit
+## Design kit
 
-### What's new
+All components, colors, icons, typography, motion, and grid elements in the kit align with the IBM Design Language.
 
-- New Sketch Cloud method of keeping your design kit in sync
-- New IBM Design Language styling
-- Improved kit structure
+### What changed
+
+- The Sketch kit is delivered with Sketch Cloud
+- New color tokens
 - Updated text styles
-- Updated spacers
+- Updated spacing
 - Icon updates
-- New color tokens
-- Updated elements page
-- New Getting Started page
 - Updated layer styles
 
-For more details on changes to the design kit, see the What's new page within the Sketch kit.
-
-### Design kit
-
-Carbon v10 includes a completely revised Sketch design kit, updated for the new IBM Design Language. All components, colors, icons, typography, motion, and grid elements in the kit are aligned with the new Design Language.
-
 #### Sketch libraries
 
-Also new in Carbon v10 is Sketch Cloud Sync for the design kit. This is a completely new way of working with the Carbon design kit. Designers can now subscribe to the design kit library via Sketch Cloud, and updates to the contents of the kit will be pushed to subscribers as they become available.
+Designers can subscribe to the design kit library via Sketch Cloud, and updates to the contents of the kit will be pushed to subscribers as they become available.
 
 #### Icons, pictograms, and the full IBM color palette
 
-To access the IBM Design Language icons, pictograms, and colors, subscribe to the IBM Design Language Kit via Sketch Cloud. **Please note that all icons in the Carbon kit are now imported from the IBM Design Language library**.
+To access the IBM Design Language icons, pictograms, and colors, subscribe to the [IBM Design Language libraries](/get-started/design/sketch#get-the-kit).
```

Comment:
It's a little confusing this appears to link to the theme kits and not the language kits. I think its an anchor link issue but I'm wondering if we should link to the resource page so its clearer: https://www.carbondesignsystem.com/resources#color-grid-and-icons